.github/workflows/release-new-version.yml

```diff
@@ -53,6 +53,29 @@ jobs:
           git: https://github.com/SanchithHegde/changelog-gh-usernames
           rev: dab6da3ff99dbbff8650c114984c4d8be5161ac8
 
+      - name: Set Git Configuration
+        shell: bash
+        run: |
+          git config --local user.name 'github-actions'
+          git config --local user.email '41898282+github-actions[bot]@users.noreply.github.com'
+
+      - name: Update Postman collection files from Postman directories
+        shell: bash
+        run: |
+          # maybe we need to move this package.json as we need it in multiple workflows
+          npm ci
```

Comment:
Wait, where is this defined?

Reply:
Here in package.json: https://github.com/juspay/hyperswitch/blob/generate-postman-collections/package.json#L7

The [corresponding lock](https://github.com/juspay/hyperswitch/blob/generate-postman-collections/package-lock.json#L11) file is generated and that snapshot is what is installed when you run `npm ci`.

Reply:
I didn't know `npm ci` was a built-in command, I thought it would require an entry in `package.json`, similar to how it's written for `npm start`.

Marking this conversation resolved.